environment.yml

```diff
@@ -0,0 +1,17 @@
+name: UPenn_Holdings_Project
```

Comment:
I think it makes sense to use the same name for the environment as the repo (`library-access`), just so the relationship is more obvious.

Reply:
Sure, no problem. Done in d035fe6.